xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererOMX.cpp

```diff
@@ -27,6 +27,7 @@
 #include "utils/GLUtils.h"
 #include "settings/MediaSettings.h"
 #include "windowing/WindowingFactory.h"
+#include "DVDCodecs/Video/DVDVideoCodecIMX.h"
```

Comment:
Yes, currently it is DVDVideoCodecOpenMax and after "DVD" has been dropped VideoCodecOpenMax remains. Sorry for my typo, you are right. It should be RendererOpenMax.